apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -182,9 +83,10 @@ const ProjectCard = React.createClass({
 
   render() {
     const { projectData, currentGallery } = this.props;
-
-    const {type, channel, name} = this.props.projectData;
-    const url = currentGallery === 'personal' ? `/projects/${type}/${channel}/edit` : `/projects/${type}/${channel}`;
+    const { type, channel } = this.props.projectData;
+    const personalGallery = currentGallery === 'personal';
+    const publicGallery = currentGallery === 'public';
```

Comment:
You have three similarly named vars, one of which is a string (currentGallery) and two of which are bools (personalGallery, publicGallery). I might change to isPersonalGallery/isPublicGallery, or leave those and change to currentGalleryType